pytest-embedded/pytest_embedded/plugin.py

```diff
@@ -142,6 +142,14 @@ def pytest_addoption(parser):
     base_group.addoption(
         '--logfile-extension', default='.log', help='set the extension format of the log files. (Default: ".log")'
     )
+    base_group.addoption(
+        '--preserve-python-tests-in-report',
```

Comment:
```python
    parser.add_argument(
        "--unity-test-report-mode",
        choices=["merge", "replace"],
        default="replace",
        help=(
            "Specify how unity test cases should be handled in the main JUnit report. "
            "'merge' will include them alongside the parent Python test case. "
            "'replace' will substitute the parent Python test case with unity test cases (default)."
        )
    )
```

how about setting it with "behavior", and emphasis "unity"?

Reply:
besides, please move this argument before line 116. now the code is in the code blocks that is for cli args which support parametrization. which is not the case for this option.